src/components/RangeSlider/components/SingleThumb/tests/SingleThumb.test.tsx

```diff
@@ -16,238 +16,256 @@ const mockProps = {
       );
-      const errorID = element.find('input').prop<string>('aria-describedby');
+      const errorID = element.find('input')!.prop('aria-describedby');
 
       expect(typeof errorID).toBe('string');
-      expect(element.find(`#${errorID}`).text()).toBe('Some error');
+      expect(element).toContainReactComponent(InlineError, {
+        message: 'Some error',
+      });
     });
 
     it('connects the input to both an error and help text', () => {
-      const element = mountWithAppProvider(
+      const element = mountWithApp(
         <SingleThumb {...mockProps} helpText="Some help" error="Some error" />,
       );
       const descriptions = element
-        .find('input')
-        .prop<string>('aria-describedby')
+        .find('input')!
+        .prop('aria-describedby')!
         .split(' ');
-
       expect(descriptions).toHaveLength(2);
-      expect(element.find(`#${descriptions[1]}`).text()).toBe('Some help');
-      expect(element.find(`#${descriptions[0]}`).text()).toBe('Some error');
+
+      expect(element).toContainReactComponent(InlineError, {
+        message: 'Some error',
+      });
+      expect(element.find('div', {id: descriptions[1]})).toContainReactText(
+        'Some help',
+      );
     });
 
     describe('prefix', () => {
       const text = 'prefix text';
 
       it('outputs the provided prefix element', () => {
-        const element = mountWithAppProvider(
+        const element = mountWithApp(
           <SingleThumb {...mockProps} prefix={<p>{text}</p>} />,
         );
-        const prefixElement = element.find('p');
+        const prefixElement = element.find('p')!;
 
-        expect(prefixElement.text()).toBe(text);
+        expect(prefixElement).toContainReactText(text);
       });
     });
   });
 
   describe('suffix', () => {
     const text = 'suffix text';
     it('outputs the provided suffix element', () => {
-      const element = mountWithAppProvider(
+      const element = mountWithApp(
         <SingleThumb {...mockProps} suffix={<p>{text}</p>} />,
       );
       const suffixElement = element.find('p');
 
-      expect(suffixElement.text()).toBe(text);
+      expect(suffixElement).toContainReactText(text);
     });
   });
 
   describe('CSS custom properties', () => {
     it('gets set on the parent element', () => {
-      const element = mountWithAppProvider(<SingleThumb {...mockProps} />);
+      const element = mountWithApp(<SingleThumb {...mockProps} />);
       const expected = {
         '--Polaris-RangeSlider-min': 10,
         '--Polaris-RangeSlider-max': 20,
         '--Polaris-RangeSlider-current': 15,
         '--Polaris-RangeSlider-progress': '50%',
         '--Polaris-RangeSlider-output-factor': '0',
       };
-      const actual = element.find('[style]').prop('style');
 
-      expect(expected).toStrictEqual(actual || {});
+      expect(element).toContainReactComponent('div', {
+        style: expect.objectContaining(expected) || {},
```

Comment:
Do you know what's going on here? Why `expected || {}` -- shouldn't it just be `expected`?

Reply:
Yeah looking back at it I think you're right, there's no reason to expect anything else.